extra-requirements.txt

```diff
@@ -36,6 +36,7 @@ docarray>=0.16.4:           core
 jina-hubble-sdk>=0.30.4:    core
 jcloud>=0.0.35:             core
 opentelemetry-api>=1.12.0:  core
+gradio:                     core
```

Comment:
I am not sure if it is good to add this dependency as a core dependency. How big and how many extra dependencies does gradio bring?

Reply:
yes, makes sense. Do you want me to add it some other type or remove it all together (and ask users to install it separately) ?

Reply:
For now lets ask users to install it, let think about it later then